src/app/modules/choice-menu/rdf-schema.d.ts

```diff
@@ -1,12 +1,14 @@
 import { ComponentSchema, json } from 'app/renpi/rdf-schema';
-import { ActionDef } from './choice-menu.component';
+import { ActionDef, ActionOptDef } from './choice-menu.component';
 
 interface ChoiceMenuActionLd extends ActionDef { }
+interface ChoiceMenuActionOptLd extends ActionOptDef { }
```

Comment:
Basically, it seems not the actions of the menu or the buttons themselves, not like the buttons assigning covered by `ActionDef`, which may be could be considered that no additional interface definition needed here.